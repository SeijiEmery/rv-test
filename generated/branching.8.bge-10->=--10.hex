0000000    63  54  b5  00  13  03  00  00  73  00  10  00                
0000014
