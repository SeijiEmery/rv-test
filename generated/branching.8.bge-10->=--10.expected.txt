R10 = 10
R6 = 1
